internal/pkg/template/workload.go

```diff
@@ -531,8 +502,13 @@ func envControllerParameters(o WorkloadOpts) []string {
 	if o.WorkloadType == "Load Balanced Web Service" {
 		parameters = append(parameters, []string{"ALBWorkloads,", "Aliases,"}...) // YAML needs the comma separator; resolved in EnvContr.
 	}
+	if o.WorkloadType == "Request-Driven Web Service" {
+		if o.Network.SubnetsType == PrivateSubnetsPlacement {
+			parameters = append(parameters, "NATWorkloads,")
+		}
+	}
 	if o.Network.SubnetsType == PrivateSubnetsPlacement {
-		parameters = append(parameters, "NATWorkloads,") // YAML needs the comma separator; resolved in EnvContr.
+		parameters = append(parameters, "NATWorkloads,")
```

Comment:
Would this result in "NATWorkloads,"  to be appended twice? maybe we don't need `if o.WorkloadType == "Request-Driven Web Service"`